stefanteixeira/demo-playwright-test
# Setup visual and a11y tests

### Description 

Setting up a sample a11y test with [axe-playwright](https://www.npmjs.com/package/axe-playwright) and a couple of visual regression tests using [happo-playwright integration](https://docs.happo.io/docs/playwright#continuous-integration). Also, configured a CI job for visual regression tests and enabled Happo app in the repository.

### How to test

- `yarn install`
- Check a11y test in `tests/e2e/create-user.e2e.test.js` (it's skipped as it will always fail, since the component has a11y issues)
- To run visual regression tests: `yarn test:visual` (you will need an account [on Happo](https://happo.io/signup) to run it locally)
- Check [Happo report](https://happo.io/a/891/p/1258/compare/26e2a50411b0d9872bd6a9ee61fb14f5a7e6a767/92182399b30428310d30026a8af386db95a7d7a7?t=added) with added screenshots

### Checklist

- [x] Branch name follows the pattern: `concise-feature-description`
- [x] PR has a descriptive name
- [x] PR branch is up-to-date with `main` (if not, rebase it)
